view/frontend/templates/search/wrapper.phtml

```diff
@@ -4,105 +4,127 @@ $addMoveToTopWidget = $configs['search']['configs']['pagination']['moveToTopWidg
 ?>
 <script type="text/template" id="wizzy-search-wrapper">
     <?= $block->getBlockHtml('formkey'); ?>
-   <div class="wizzy-search-wrapper {{#inOnCategoryPage}}wizzy-category-page{{/inOnCategoryPage}}
+    <div class="wizzy-search-wrapper 
+        {{#inOnCategoryPage}}wizzy-category-page{{/inOnCategoryPage}}
+        {{#displayTileView}}display-tile-view{{/displayTileView}}
         {{#hasFacets}}has-facets{{/hasFacets}}
+        {{#hasToDisplayFiltersAtTop}}has-filters-button-at-top{{/hasToDisplayFiltersAtTop}}
         {{#hasLeftFacets}}has-left-facets{{/hasLeftFacets}}
         {{#hasTopFacets}}has-top-facets{{/hasTopFacets}}
         {{#displayFacetsInDrawer}}has-facets-in-drawer facets-drawer-on-{{facetDrawerPosition}}
-        {{/displayFacetsInDrawer}}">
-
-      <div class="wizzy-search-summary-wrapper">
-         {{#summary}}
-            {{{summary}}}
-         {{/summary}}
-      </div>
-
-      <div class="wizzy-search-results-wrapper">
-          {{#hasFacets}}
-                <a href="#" class="wizzy-filters-mobile-entry {{^sort}}only-filters{{/sort}}">Filters</a>
-          {{/hasFacets}}
-
-         {{#hasLeftFacets}}
-             <div class="wizzy-search-filters-left">
-
-                 <div class="wizzy-filters-bg"></div>
-                 <div class="wizzy-filters-close-btn"></div>
-                 <div class="wizzy-filters-header">
-                     <p class="header-title">Filters</p>
-                     {{#hasFilters}}
-                        <a href="#" class="wizzy-filters-clear-all">Clear All</a>
-                     {{/hasFilters}}
-                 </div>
-
-                 <div class="wizzy-search-filters-left-wrapper">
-                     {{#leftFacets}}
-                        {{{html}}}
-                     {{/leftFacets}}
-
-                     <div class="wizzy-search-filters-left-mobile-extra">
-                     </div><!-- ending of wizzy-search-filters-left-mobile-extra -->
-                 </div>
-
-             </div><!-- ending of wizzy-search-filters-->
-          {{/hasLeftFacets}}
-
-         <div class="wizzy-search-results-container">
-
-            <div class="wizzy-search-filters-top">
-            {{#displayFacetsInDrawer}}
-                    <div class="wizzy-search-filters-drawer">
-                        <div class="wizzy-search-filters-drawer-button">
-                            <a href="#" class="wizzy-filters-desktop-entry">
-                                <span class="wizzy-filters-button"></span>
-                                <span>Filters</span>
-                            </a>
-                        </div>
+        {{/displayFacetsInDrawer}}"
```

Comment:
Too many changes due to intent. Keep modifications in a way to have minimal changes.